webview/src/experiments/components/App.test.tsx

```diff
@@ -885,24 +893,12 @@ describe('App', () => {
       advanceTimersByTime(100)
       expect(screen.getAllByRole('menuitem')).toHaveLength(10)
 
-      fireEvent.click(window, { bubbles: true })
-      advanceTimersByTime(100)
-      expect(screen.queryAllByRole('menuitem')).toHaveLength(0)
-    })
-
-    it('should be removed with a left click on a different row', () => {
```

Comment:
[F] This behaviour now relies `onClickOutside` which won't work in an environment with 0 height and width. The behaviour is included in the demo.

Reply:
comment is outdated